tests/app/Models/LogDAOTest.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+use PHPUnit\Framework\TestCase;
+
+class LogDAOTest extends TestCase {
+	private $logDAO;
+	protected function setUp(): void {
+		$this->logDAO = new FreshRSS_LogDAO();
+		file_put_contents(ADMIN_LOG, '[Wed, 08 Feb 2023 15:35:05 +0000] [notice] --- Migration 2019_12_22_FooBar: OK');
```

Comment:
I am not sure I like this whole test. In particular, it overrides the local admin log, if I am not mistaken.

Reply:
This is the principle of a unit test. I'm not hijacking the original functionality. I'm testing it from start to finish

Reply:
Using the same principles, you could also drop the whole database, which I do not believe would be desirable.
This has too much side effects, as nothing says that the system is not in production.

Reply:
We never run the tests in production. 
I was inspired by what was done for MigratorTest.php which also creates log files

Reply:
You might not run tests in production, but people / users might. Yes, the Migrator stuff (which is not used anyway at the moment) also needs some rework, as it pollutes the disk with plenty of files which are not properly cleaned.

Reply:
I added a function to clean up after the test

Reply:
I suggest you use a temporary file instead of `ADMIN_LOG`, which might contain some information

Reply:
I did what you asked me to do

Reply:
Maybe you could define envvars in the phpunit config file. This way, it works both in production environment and test environment

Reply:
Going this way will force us to use envvars for configuration. But I think it will be interesting. We can still use current values as default values when using envvars but this will ease configuration for powerusers.